craft_cli/dispatcher.py

```diff
@@ -245,7 +245,7 @@ def load_command(self, app_config: Any) -> BaseCommand:
         parser = _CustomArgumentParser(self._help_builder, prog=self._loaded_command.name)
         self._loaded_command.fill_parser(parser)
         self._parsed_command_args = parser.parse_args(self._command_args)
-        emit.trace(f"Command parsed sysargs: {self._parsed_command_args}")
+        emit.debug(f"Command parsed sysargs: {self._parsed_command_args}")
```

Comment:
I know that I was the one that suggested this but now I'm afraid of leaking stuff that we shouldn't leak from the command line. For example, what if someone calls `snapcraft pack  --ua-token <token>`?

Reply:
This is a general problem with CLI stuff. It gets logged in your bash history too.

Secrets like that really should be passed through environment variables.